grayskull/license/discovery.py

```diff
@@ -344,6 +345,10 @@ def get_license_type(path_license: str, default: Optional[str] = None) -> Option
     best_match = process.extract(
         license_content, licenses_text, scorer=token_sort_ratio
     )
+    print(
```

Comment:
can you replace the print by
```suggestion
    print_msg(
```
please?